lms/templates/dashboard.html

```diff
@@ -99,6 +99,13 @@
         banner.showMessage(${recovery_email_activation_message | n, dump_js_escaped_json})
       </%static:require_module>
   % endif
+  % if enterprise_learner_portal_enabled_message:
+      <%static:require_module module_name="js/views/message_banner" class_name="MessageBannerView">
+        var banner = new MessageBannerView({urgency: 'low', type: 'warning', isRecoveryEmailMsg: true});
```

Comment:
What does `isRecoveryEmailMsg: true` mean here? Should it be `false` (or removed?) since this banner doesn't pertain to the recovery email?

Reply:
oo good catch, slipped through the copy/pasta cracks

Reply:
With Joe I reviewed the styling of the message with this flag vs without it and Joe likes the look of the banner better when this flag is enabled. "[It] catches the eye more"